staging/src/k8s.io/client-go/util/workqueue/delaying_queue_test.go

```diff
@@ -27,24 +28,44 @@ import (
 	testingclock "k8s.io/utils/clock/testing"
 )
 
+const (
+	DefaultWaitTimeout = 2 * time.Second      // The default duration we will wait before giving up when polling for a result.
+	DefaultWaitStep    = 5 * time.Millisecond // The default pause between attempted polls when polling for a result.
+)
+
+// InspectableDelaying has methods which allow a client to inspect the state of items the 'waiting' part of the delaying queue.
+// Our delayingType implements this interface.
+type InspectableDelaying interface {
```

Comment:
PTAL also at these resolved answers 
- https://github.com/kubernetes/kubernetes/pull/112328#discussion_r1154767849
- https://github.com/kubernetes/kubernetes/pull/112328#discussion_r1154777396

Reply:
Ok making private - I kinda thought that this would be useful in client tests but now I'd be happy to give us an opportunity to make use of the finer control.